parse.ts

```diff
@@ -58,23 +65,32 @@ export function parseWithValibot<
 
       const resolveResult = (
         result: SafeParseResult<Schema>,
-      ):
-        | { value: InferOutput<Schema> }
-        | { error: Record<string, string[]> } => {
+      ): { value: InferOutput<Schema> } | { error: ErrorType } => {
         if (result.success) {
           return {
             value: result.output,
           };
         }
 
         return {
-          error: result.issues.reduce<Record<string, string[]>>((result, e) => {
+          error: result.issues.reduce<ErrorType>((result, e) => {
+            if (
+              result === null ||
+              e.message === conformValibotMessage.VALIDATION_UNDEFINED
+            ) {
+              return null;
```

Comment:
If error field is null, Conform [stop reporting errors](https://github.com/edmundhung/conform/blob/b0fb91dbab474c7a150d5f56cb71dbb92888779c/packages/conform-dom/form.ts#L814-L821) and [follow browser's built-in form submission process](https://github.com/edmundhung/conform/blob/b0fb91dbab474c7a150d5f56cb71dbb92888779c/packages/conform-react/context.tsx#L493-L497) (don't call `event.stopPropagation` for `submit` event).

Reply:
Same as the Zod version 👍 

https://github.com/edmundhung/conform/blob/b0fb91dbab474c7a150d5f56cb71dbb92888779c/packages/conform-zod/parse.ts#L28-L29